### PR TITLE
[#134795163]get_framework_suppliers: make inclusion of declaration section optional

### PR DESCRIPTION
**Pull request by risicle**
This seems like the most sensible way to deal with the issue in story https://www.pivotaltracker.com/story/show/134795163

:man_shrugging:

### CODE REVIEW

File: app/main/views/frameworks.py [hunk 1]
```diff
@@ -213,14 +214,14 @@ def get_framework_suppliers(framework_slug):
     ).first_or_404()
 
     # we're going to need an "alias" for the current_framework_agreement join
-    # so that we can refer to it in later clauses
+    # so that we can refer to it in later clausesris
```
Comment (TheDoubleK) [inline app/main/views/frameworks.py]: Rlly?

File: app/main/views/frameworks.py [hunk 2]
```diff
@@ -251,7 +252,10 @@ def get_framework_suppliers(framework_slug):
         )
 
     return jsonify(supplierFrameworks=[
-        supplier_framework.serialize(with_users=False) for supplier_framework in supplier_frameworks
+        supplier_framework.serialize(
+            with_users=False,
+            with_declaration=convert_to_boolean(request.args.get("with_declarations", "true")),
```
Comment (TheDoubleK) [inline app/main/views/frameworks.py]: Would it be worth pulling the `convert_to_boolean` out before this so it doesn't get called for every supplier_framework?

File: app/main/views/frameworks.py (hunk 1 above)
Comment (risicle) [inline app/main/views/frameworks.py]: Shit! I explicitly meant to yank that out...

File: app/main/views/frameworks.py (hunk 2 above)
Comment (risicle) [inline app/main/views/frameworks.py]: good idea

**Pull request merged**